src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useMemo } from 'react';
+import { Todo } from '../../types/Todo';
+import { FILTER_OPTIONS } from '../../utils/constants';
+
+type Props = {
+  todos: Todo[],
+  todosToRender: Todo[]
+  setTodosToRender: (arr: Todo[]) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  todosToRender,
+  setTodosToRender,
+}) => {
+  const [selectedButton, setSelectedButton] = useState('All');
```

Comment:
it will be great introduce `enum FilterType` and use it everywhere
```suggestion
  const [selectedButton, setSelectedButton] = useState(FilterType.All);
```